deploy/cr.yaml

```diff
@@ -6,7 +6,12 @@ metadata:
     - percona.com/delete-mysql-pods-in-order
     # - percona.com/delete-ssl
 spec:
-  allowUnsafeConfigurations: false
+  unsafeFlags:
```

Comment:
I think we need to comment on this section as we did for PSMDB https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/cr.yaml#L30-L35

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/668/commits/71d74cd32882ec94d614aa0e660b5f7fa5535bc4